week-03/main.js

```diff
@@ -43,10 +60,33 @@ var list = [1, 2, 3, 4, 5];
 
 var list = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
+var odd= list.filter((function() {
```

Comment:
Filter takes a callback function, which means it internally applies that function to every element in the array. It iterates automatically, therefore you shouldn't call a `for` loop inside that function.